src/guide/typescript-support.md

```diff
@@ -133,6 +133,45 @@ const Component = defineComponent({
 })
 ```
 
+### Augmenting Types for Use with Plugins or Declarations by User
```

Comment:
I'm wondering whether something like this might give a clearer sense of what follows?

```suggestion
### Augmenting Types for `globalProperties`
```

If I've understood correctly then that's what this section is really about.

Reply:
Yes, at first I used "Use with Plugins" because the [documentation of Vue 2.x](https://vuejs.org/v2/guide/typescript.html#Augmenting-Types-for-Use-with-Plugins) uses it, so I copied from the old document. Next, I realized that not only plugins, users can also define some properties (This is exactly what I realized that the documentation was not clear about this point), so I add "or Declarations by User".

In fact, I did not realize the essential meaning of the problem before, whether it is a user or a plug-in, they are all "add" global properties.